test/fft_utest.cpp

```diff
@@ -140,6 +144,68 @@ class FftTest : public ::testing::Test {
             ASSERT_EQ(ifft_1, v);
         }
     }
+
+    /**
+     * convert a number into a vector of digits padded with zeros
+     *
+     * @param num
+     *
+     * @return
+     */
+    quadiron::vec::Vector<T>*
+    _convert_string2vec(const gf::Field<T>& gf, int n, char num[])
+    {
+        int i;
+        quadiron::vec::Vector<T>* vec = new quadiron::vec::Vector<T>(gf, n);
+        int len = strlen(num);
+        for (i = 0; i < len; i++) {
+            vec->set(i, num[len - i - 1] - '0');
+        }
+        for (; i < n; i++) {
+            vec->set(i, 0);
+        }
+        return vec;
+    }
+
+    /** compute sum of 2 large numbers in string representation
```

Comment:
`/** Compute sum of 2 large numbers in string representation.`

(proper sentence: add capital and period)